packages/roosterjs-editor-plugins/lib/plugins/ImageEdit/imageEditors/Resizer.ts

```diff
@@ -108,59 +110,117 @@ export function doubleCheckResize(
  * Get HTML for resize handles at the corners
  */
 export function getCornerResizeHTML({
+    editInfo: editInfo,
     borderColor: resizeBorderColor,
+
+    circularHandlers: circularHandlers,
 }: ImageHtmlOptions): CreateElementData[] {
     const result: CreateElementData[] = [];
+
     Xs.forEach(x =>
         Ys.forEach(y =>
             result.push(
-                (x == '') == (y == '') ? getResizeHandleHTML(x, y, resizeBorderColor) : null
+                (x == '') == (y == '')
+                    ? getResizeHandleHTML(x, y, resizeBorderColor, circularHandlers)
+                    : null
             )
         )
     );
     return result;
 }
 
+function imageArea(width: number, height: number): number {
+    return width * height;
+}
+
 /**
  * @internal
  * Get HTML for resize handles on the sides
  */
 export function getSideResizeHTML({
+    editInfo: editInfo,
     borderColor: resizeBorderColor,
+    sizeAdaptiveHandlers: sizeAdaptiveHandlers,
+    circularHandlers: circularHandlers,
 }: ImageHtmlOptions): CreateElementData[] {
+    const { widthPx, heightPx } = editInfo;
+    if (sizeAdaptiveHandlers && widthPx && heightPx && imageArea(widthPx, heightPx) < 10000) {
+        return;
```

Comment:
return what?

Reply:
Why return when sizeAdaptiveHandlers is true?

Reply:
when sizeAdaptiveHandlers is true, we do not need to create the side handles, only the corners.